import-export-cli/cmd/mg/delete.go

```diff
@@ -0,0 +1,50 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+const deleteCmdLiteral = "delete"
+const deleteCmdShortDesc = "Delete an API in Microgateway"
+const deleteCmdLongDesc = `Delete an API by specifying name, version, host, username 
+and optionally vhost by specifying the flags (--name (-n), --version (-v), --host (-c), 
+--username (-u), and optionally --vhost (-t). Note: The password can be included 
+via the flag --password (-p) or entered at the prompt.`
```

Comment:
```suggestion
const deleteCmdLongDesc = `Delete an API in Microgateway`
```

Reply:
Shall we change like this? We do not need to specify about the flag usages here. You can add them to deleteCmdExamples variable.

Reply:
Then wouldn't we be mixing up the a "readable" part and a "usable" part. One might refer to the "Long Description" to know about the command's functionality together with important things to note. If they already know all the details and only want a sample commands just so that they can copy paste, replace the necessary with their own data, and execute it, they can refer to the examples section.

Reply:
Shall we do like this [1] what we have done for "apictl k8s delete"?
You can just mention
const deleteCmdLongDesc = "Delete resources related to Microgateway by specifying name, version, host, username and optionally vhost"
or something which gives the above meaning.

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/cmd/k8s/delete.go

Reply:
Also, the "NOTE: The 4 flags (--name (-n), --version (-v), --host (-c) and --username (-u)) are mandatory. (The password can be included via the flag --password (-p) or entered at the prompt)" should be moved to **deleteCmdExamples**.

For example; Check "apictl add env"
![image](https://user-images.githubusercontent.com/25246848/108015453-4c904400-7036-11eb-9bed-29bf31ad7a73.png)

Reply:
I see. Thanks for the screenshot. I'll remove the flags from the "Long Description", keep only what each meant, and move the NOTE to examples.

Reply:
Thanks